Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2318,6 +2360,169 @@ def add_custom_fields(params):
+    if not isinstance(items, list):
+        items_list = [items]
+    else:
+        items_list = items
+    if len(items_list) == 0:
+        return CommandResults(
+            outputs_prefix="ServiceNow.Tasks",
+            readable_output='No items were found.',
+            raw_response=result
+        )
+
+    mapped_items = []
+    for item in items_list:
+        mapped_items.append(get_tasks_from_co_human_readable(item))
+
+    headers = ['ID', 'Name', 'State', 'Description']
+    human_readable = tableToMarkdown('ServiceNow Catalog Items', mapped_items, headers=headers,
+                                     removeNull=True, headerTransform=pascalToSpace)
+    entry_context = {'ServiceNow.Tasks(val.ID===obj.ID)': createContext(mapped_items, removeNull=True)}
+
+    return CommandResults(
+        outputs_prefix="ServiceNow.Tasks",
+        outputs=entry_context,
+        readable_output=human_readable,
+        raw_response=result
+    )
+
+
+def create_co_from_template_command(client: Client, args: dict) -> CommandResults:
+    """Create a change request from a template.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        Demisto Outputs.
+    """
+
+    template = args.get('template', "")
+    result = client.create_co_from_template(template)
+    if not result or 'result' not in result:
+        raise Exception('Unable to retrieve response.')
+    ticket = result['result']
+    human_readable_table = get_co_human_readable(tickets=ticket, ticket_type='change_request')
+    headers = ['System ID', 'Number', 'Impact', 'Urgency', 'Severity', 'Priority', 'State', 'Approval',
+               'Created On', 'Created By', 'Active', 'Close Notes', 'Close Code', 'Description', 'Opened At',
+               'Due Date', 'Resolved By', 'Resolved At', 'SLA Due', 'Short Description', 'Additional Comments']
+    human_readable = tableToMarkdown('ServiceNow ticket was created successfully.', t=human_readable_table,
+                                     headers=headers, removeNull=True)
+    created_ticket_context = get_ticket_context(ticket)
+    entry_context = {
+        'Ticket(val.ID===obj.ID)': created_ticket_context,
+        'ServiceNow.Ticket(val.ID===obj.ID)': created_ticket_context
+    }
+    return CommandResults(
+        outputs_prefix="ServiceNow.Ticket",
+        outputs=entry_context,
+        readable_output=human_readable,
+        raw_response=result
+    )
+
+
+def get_co_human_readable(tickets, ticket_type: str, additional_fields: list = None) -> list:
+    """Get co human readable.
+
+    Args:
+        tickets: tickets data. in the form of a dict or a list of dict.
+        ticket_type: ticket type.
+        additional_fields: additional fields to extract from the ticket
+
+    Returns:
+        ticket human readable.
+    """
+    if not isinstance(tickets, list):
+        tickets = [tickets]
+
+    items = []
+    for ticket in tickets:
```

Comment:
why is this handling an array? isn't `create ticket` create only one at a time?

Reply:
from the api documentation:
GET /sn_chg_rest/change/standard
Retrieves one or more standard change requests based on the specified criteria.
https://developer.servicenow.com/dev.do#!/reference/api/rome/rest/change-management-api#changemgmt-POST-emerg-create-chng-req

Reply:
You are right, found in the documentation:
POST /sn_chg_rest/change/standard/{standard_change_template_id}
Creates one standard change request based on an existing standard change template as specified by the passed-in template sys_id. Multiple standard change request creations within a single call is not supported.

The customer sent the code like this, because of it I thought it is supported, will change it